main.go

```diff
@@ -4,12 +4,15 @@ import (
 	"fmt"
 	"log"
 	"os"
+	"strconv"
```

Comment:
Suggestion: Consider moving strconv and strings imports together for organization purposes.